.gitattributes

```diff
@@ -1,6 +1,5 @@
 package-lock.json lockable
 * text=auto eol=lf
-*.pbxproj -text
```

Comment:
Why do you need that change?

Reply:
That's in the diff, weird though for an update

Reply:
Fair enough.